### PR TITLE
Updated the short-term OSG integration documentation.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: osg/short-term.md [hunk 1]
```diff
@@ -169,37 +175,23 @@ tool.
...
-    "irods_port": <port-number>,
-    "irods_zone_name": ""
-}
-```
-
-For the time being, the zone name should always be equal to the empty string. The rest of the settings are extracted
-from the job configuration file. After creating this file, `iget` and `iput` can be used in conjunction with the iRODS
-tickets to retrieve files from or push files to iRODS.
+As long as you're using version 4.x of the icommands and not using custom iRODS credentials, initializing the iRODS
+connection is very easy. The file, `config.json` is made to be compatible with the file that the icommands use to
```
Comment (yeea) [inline osg/short-term.md]: ~I'd either drop the comma after file or add another comma after config.json.~ Just kidding, I see there's another file being referred to, so I'd lop off the leading couple words:
```suggestion
connection is very easy. `config.json` is made to be compatible with the file that the icommands use to
```

File: osg/short-term.md [hunk 2]
```diff
@@ -169,37 +175,23 @@ tool.
...
-If you're using custom iRODS credentials with a password then (assuming you're not using Jargon) `iinit` must be called
-before using any of the iRODS icommands. `iinit` only runs in interactive mode, so it will be necessary to send the
-password to `iinit` using redirection. Using custom iRODS credentials is expected to be rare, so precise instructions
-for running `iinit` are beyond the scope of this documentation. Similarly, using Jargon is beyond the scope of this
-documentation.
+If you're using custom iRODS credentials or an older version of the icommands then (assuming you're not using Jargon)
+`iinit` must be called before using any of the iRODS icommands. `iinit` only runs in interactive mode, so it will be
+necessary to send the password to `iinit` using redirection. Using older icommands versions or custom iRODS credentials
+is expected to be rare, so precise instructions for running `iinit` are beyond the scope of this
+documentation. Similarly, using Jargon is beyond the scope of this documentation.
```
Comment (yeea) [inline osg/short-term.md:187]:
```suggestion
documentation. Similarly, using Jargon is beyond the scope of this documentation, and perhaps, even, hope.
```

**Review by yeea — APPROVED** [review]
Not that I'm very deep into this stuff, but docs are clear and look good to me!

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**